### PR TITLE
DBG - testUsed UDisks2 timeout

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: test/verify/check-storage-used [hunk 1]
```diff
@@ -24,7 +24,7 @@ import testlib
 @testlib.nondestructive
 class TestStorageUsed(storagelib.StorageCase):
 
-    def testUsed(self):
+    def checkUsed(self):
```
Comment (martinpitt) [inline test/verify/check-storage-used:27]: There's no need for this.

File: test/verify/check-storage-used [hunk 2]
```diff
@@ -89,12 +89,37 @@ ExecStart=/usr/bin/sleep infinity
...
+        except:
+            print(m.execute("lsblk; udevadm info /dev/sda"))
+            raise
 
         m.execute("! systemctl --quiet is-active keep-mnt-busy")
 
         self.content_row_wait_in_col(1, 0, "Free space")
 
+    def testUsed1(self):
+        self.checkUsed()
```
Comment (martinpitt) [inline test/verify/check-storage-used:103]: Just call `self.testUsed()` in the amplifications.

File: test/verify/check-storage-used (hunk 1 above)
Comment (mvollmer) [reply test/verify/check-storage-used:27]: Yes yes yes... :-)

**Comment by mvollmer** [comment]
My hunch right now is that we will have to retry the dialog. But let's first try to collect some data on what is actually going wrong.

**Comment by mvollmer** [comment]
On rhos, testUsed seems reliably green.

**Comment by mvollmer** [comment]
UDisks2 locks the LUKS device during teardown, waits for the cleartext device to disappear, and immediately wipes the partition table, which succeeds (because nothing has /dev/sda open at the time). Then UDisks2 does a BLKRRPART ioctl which fails with EBUSY: something must still have /dev/sda1 open at that time. Maybe udev? But there is no uevent on /dev/sda1 during all of this. Hmm.

**Pull request closed without merging**